pretext/ExtraTopics/DataScience.ptx

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-datascience">
+  <title>Data Science</title>
+
```

Comment:
Mentioning that data science is a multidisciplinary field combining CS, math, and other domains I think would be good here before immediately diving into the steps. I'll likely want to expand this section in other ways too, but I can do that myself.